tools/specification_parser/test_specification.md

```diff
@@ -4,27 +4,27 @@
 
 Some content.
 
-##### Requirement Some requirement
+#### Requirement Some requirement
```

Comment:
I would have expected to see some sort of anchor link in this to show off the problem you mentioned above.

Reply:
Here is an example of what was breaking before the parser fix: https://github.com/open-feature/spec/pull/171/files#diff-300118ed122b34e94bd16e8714f09eb91547e48b20e087d4b5ad2847698581d3R112